var/spack/repos/builtin/packages/qt/package.py

```diff
@@ -724,17 +718,28 @@ def configure(self, spec, prefix):
 
         if '~opengl' in spec:
             config_args.extend(['-skip', 'multimedia'])
-            if version >= Version('5.10'):
-                config_args.extend([
-                    '-skip', 'webglplugin',
-                    '-skip', 'qt3d',
-                ])
+            config_args.extend(['-skip', 'qt3d'])
+
+            if '@5.10:' in spec:
+                config_args.extend(['-skip','webglplugin'])
 
-            if version >= Version('5.14'):
+            if '@5.14:' in spec:
```

Comment:
If you're going to change how version comparison is written in some parts of a function unrelated to the change set, maybe update the rest of the version comparisons in the function for consistency?